picard/ui/logview.py

```diff
@@ -271,7 +276,7 @@ def _clear_log_do(self):
         reply = QtWidgets.QMessageBox.question(
             self,
             _("Clear Log"),
-            _("Are you sure you want to clear the log?"),
+            ("Are you sure you want to clear the log?"),
```

Comment:
Why remove i18n?

Reply:
> Why remove i18n?

Is "because i suck" an acceptable answer ? ;)
Fixed and rebased, thanks for spotting this typo.